Bond/Extensions/OSX/NSTableView+Bond.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  The MIT License (MIT)
+//
+//  Copyright (c) 2015 Michail Pishchagin (@mblsha)
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import Cocoa
+
+public protocol BNDTableViewDelegate {
+  func createCell(row: Int, array: AnyObject, tableView: NSTableView) -> NSTableCellView
```

Comment:
If I do that, I get an error: `Protocol 'BNDTableViewDelegate' can only be used as a generic constraint because it has Self or associated type requirements` at `private var delegate: BNDTableViewDelegate?` line.

And I can't specialize it either by appending `<T>` to the end: `Cannot specialize non-generic type 'BNDTableViewDelegate'`.

Am I missing something? Is there a better way to handle this?